### ISSUE TITLE
bug(dockerfile): apt-get Missing '-y' To Avoid Manual Input`

### ISSUE THREAD

**Issue by SpikePy** [Checkmarx/kics#7053]
When linting a Dockerfile that has an `apt-get install` command I get the error `77783205-c4ca-4f80-bb80-c777f267c547 # APT-GET Missing '-y' To Avoid Manual Input`.  This is a false positive when using `apt-get --quiet --quiet install sl` or  `apt-get --yes install sl`

### Expected Behavior

- No error is shown when using `apt get --quiet --quiet` as it implies `-y`/`--yes`
- No error is shown when using `apt get --yes` which is the long form of `-y`

### Actual Behavior

```
[Severity: LOW, Query description: Check if apt-get calls use the flag -y to avoid user manual input.] Problem found on 'docker/local-toolbox/Dockerfile_test' file in line 2. Expected value: {{RUN apt-get --yes install sl}} sould avoid manual input. Actual value: {{RUN apt-get --yes install sl}} doesn't avoid manual input.
```

**Comment by kicsbot** [comment]
Please, follow the guideline for an issue title: 

For **bug**:

`bug(<scope>): <title starting with lowercase letter>`

For **query**:

`query(<platform>): <title starting with lowercase letter>`

For **feature request**:

`feat(<scope>): <title starting with lowercase letter>`

Thank you!
*KICS Team*

**Comment by kicsbot** [comment]
Please, follow the guideline for an issue title: 

For **bug**:

`bug(<scope>): <title starting with lowercase letter>`

For **query**:

`query(<platform>): <title starting with lowercase letter>`

For **feature request**:

`feat(<scope>): <title starting with lowercase letter>`

Thank you!
*KICS Team*

**Comment by kicsbot** [comment]
Deleting comment, please refresh the page...

**Comment by gabriel-cx** [comment]
Hi @SpikePy ,

Thank you for your inputs!
Our internal AppSec team soon will check it.
We will keep you updated.

(APPSEC-2556)
